net/mdns-repeater/src/opnsense/service/templates/OPNsense/mdnsrepeater/mdnsrepeater

```diff
@@ -0,0 +1,8 @@
+{% if helpers.exists('OPNsense.mdnsrepeater.enabled') and OPNsense.mdnsrepeater.enabled == '1' %}
+{% from 'OPNsense/Macros/interface.macro' import physical_interface %}
+mdns_repeater_enable="YES"
+{% set osifnames = OPNsense.mdnsrepeater.interfaces.split(',') %}
+mdns_repeater_interfaces="{% for i in osifnames %}{{ physical_interface(i) }} {% endfor %}"
```

Comment:
join() is a bit more elegant, see:

https://github.com/opnsense/core/blob/master/src/opnsense/service/templates/OPNsense/IDS/rc.conf.d#L16